recipe/meta.yaml

```diff
@@ -14,17 +14,18 @@ source:
 build:
   noarch: python
   number: 0
-  script: "{{ PYTHON }} -m pip install . -vv"
+  script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
   host:
     - python >=3.8
-    - pip 
+    - pip
     - setuptools
   run:
     - python >=3.8
     - matplotlib-base >=3.3.3
-    - pytorch ==1.7.1
+    - loguru >=0.5.3
+    - pytorch ==1.9
```

Comment:
Becuase you have this as a run dependency, and no c libraries, i think you can relax this pinning. What happens if you leave it at `>=1.7.1`?